pheweb/file_utils.py

```diff
@@ -51,10 +51,9 @@ def get_cacheable_file_location(default_relative_dir, basename):
     "unanno": get_generated_path("sites/sites-unannotated.tsv"),
     "sites-rsids": get_generated_path("sites/sites-rsids.tsv"),
     "sites": get_generated_path("sites/sites.tsv"),
-    "best-phenos-by-gene": get_generated_path("best-phenos-by-gene.json"),
     "cpra-to-rsids-trie": get_generated_path("sites/cpra_to_rsids_trie.marisa"),
     "rsid-to-cpra-trie": get_generated_path("sites/rsid_to_cpra_trie.marisa"),
-    "matrix": get_generated_path("matrix.tsv.gz"),
+    "matrix": get_generated_path("finngen_R10_sumstats_p0.05.af.tsv.gz"),
```

Comment:
This should be a generally named file, something like long.tsv.gz - these filenames are the same for all pheweb instances (we want to have the same codebase for R11, R12, userresults etc.)